roles/uperf/templates/service.yml.j2

```diff
@@ -1,27 +1,46 @@
 ---
-kind: Service
 apiVersion: v1
-metadata:
-  name: uperf-service-{{ item }}-{{ trunc_uuid }}
-  namespace: '{{ operator_namespace }}'
-  labels:
-    app: uperf-bench-server-{{ item }}-{{ trunc_uuid }}
-    type: uperf-bench-server-{{ trunc_uuid }}
-spec:
-  selector:
-    app: uperf-bench-server-{{ item }}-{{ trunc_uuid }}
-  ports:
-  - name: uperf
-    port: 20000
-    targetPort: 20000
-    protocol: TCP
+kind: List
+metadata: {}
+items:
+{% macro job_template(item, node_idx_item='') %}
+  - kind: Service
+    apiVersion: v1
+    metadata:
+      #name: uperf-service-{{ item }}-{{ trunc_uuid }}
+      name:  uperf-service-{{worker_node_list[ node_idx_item ] }}-{{ item }}-{{ trunc_uuid }}
```

Comment:
Similar to my comment above, I wonder if we can do something like: worker_node_list[node_idx_item].split('.')[0]